cmd/typed-errors.go

```diff
@@ -38,6 +38,9 @@ var errSizeUnspecified = errors.New("Data size is unspecified")
 // When upload object size is greater than 5G in a single PUT/POST operation.
 var errDataTooLarge = errors.New("Object size larger than allowed limit")
 
+// When policy object size is greater than 20K
+var errPolicyTooLarge = errors.New("Policy object size larger than allowed limit")
```

Comment:
policy object size is not correct, it should be just policy document size

Reply:
You don't need this change in this PR since this error is not used anywhere....

Reply:
Deleted